### ISSUE TITLE
ServerError missing 1 required positional argument: 'message'

### ISSUE THREAD

**Issue by IlyaSemenov** [LonamiWebs/Telethon#127]
```
  File "/root/.pyenv/versions/foo/lib/python3.5/site-packages/Telethon-0.11-py3.5.egg/telethon/errors/__init__.py", line 41, in rpc_message_to_error
TypeError: __init__() missing 1 required positional argument: 'message'
```

You can see that on line 41, a `ServerError` is constructed without parameters:

```python
    elif code == 500:
        return ServerError()
```

However according to the class definition, it has a required argument `message`:

```python
class ServerError(RPCError):
...
    def __init__(self, message):
```

**Comment by Lonami** [comment]
Shoot, missed that and couldn't test. It's as simple as passing the parameter `message` when constructing it though.

**Comment by Lonami** [comment]
Fixed on be33ae4e800619e0c50ef9dd7ce5e135e2ebb54b, sorry again.
